active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,143 @@
+
+## Examples
+A Breadcrumb control with 3 nodes:
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+
+A Breadcrumb "crumbling" with 3 visible nodes:
+
+![Breadcrumb crumbling with 3 visible nodes](images/Breadcrumb_crumbling.PNG)
+
+A Breadcrumb control with the last node truncated:
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+
+The following example creates a simple Breadcrumb control with three nodes:
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+
+XAML
+~~~~ 
+     <muxc: Breadcrumb x:Name="DefaultBreadcrumb"/> 
+~~~~
+
+C#
+~~~~ 
+     DefaultBreadcrumb.ItemsSource = new string[]{ "Node", "Node" , "Node"}
+~~~~
+
+More examples with a GIF of crumbling behavior to be added.
+
+## Breadcrumb member notes
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| DropDownItemTemplate | Gets or sets the template to display the drop-down of nodes | 
+| ItemsSource | Gets or sets the content of the Breadcrumb | null |
+| ItemTemplate | Gets or sets the template to display an item|  | 
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  | N/A |
+
+Spec note: the V2 of Breadcrumb will add flyouts from chevrons to view children of a node.
+Additional properties will be added to the API to enable this functionality. 
+
+# API Details
+
+```cs
+runtimeclass BreadcrumbItemClickedEventArgs
+{
+    Object Item { get; };
+}
+
+unsealed runtimeclass BreadcrumbItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbItem();
+}
+
+unsealed runtimeclass BreadcrumbLayout : NonVirtualizingLayout
+{
+    BreadcrumbLayout();
+}
+
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass Breadcrumb : Windows.UI.Xaml.Controls.Control
+{
+    Breadcrumb();
+
+    Object DropdownItemTemplate{ get; set; };
+    Object ItemsSource{ get; set; };
+    Object ItemTemplate{ get; set; };
+
+    event Windows.Foundation.TypedEventHandler<Breadcrumb, BreadcrumbItemClickedEventArgs> ItemClicked;
+
+    static Windows.UI.Xaml.DependencyProperty DropdownItemTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemsSourceProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemTemplateProperty{ get; };
+}
+```
+
+# Appendix
+
+## UI Automation Pattern
+
+Breadcrumb will use the
+[InvokePattern](!https://docs.microsoft.com/dotnet/framework/ui-automation/implementing-the-ui-automation-invoke-control-pattern). The flyouts use the [ExpandCollapsePattern](https://docs.microsoft.com/dotnet/api/system.windows.automation.expandcollapsepattern).
+
+## Keyboard
+
+* The user can tab into the Breadcrumb and tab out.
+The first tab in gives focus to the first (highest-level) node.
+Once the Breadcrumb has focus, the user can use the left/right arrows to navigate from node to node.
+* If the Breadcrumb is crumbled, down/up will open the flyout from
```

Comment:
discussion: only invoking (space or enter) will open the flyout from the ellipsis.